.gitignore

```diff
@@ -21,4 +21,4 @@ debug
 /mac_install.log
 /win_install.log
 /linux_install.log
-/src/default_excludes.mk
\ No newline at end of file
+/src/default_excludes.mk
```

Comment:
why is this here as there are no changes. generally we do not accept changes to it.